speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h

```diff
@@ -268,12 +271,35 @@ typedef struct CheckBoxInfo
     void        *userData;      /**< Associated user data */
 } tCheckBoxInfo;
 
+// SIMULATED DRIVING ASSISTANCE CHANGE: Added radiobutton information
+/// @brief Radio-button call-back information
+typedef struct RadioButtonInfo
+{
+    bool  checked;  /**< Radio-button state */
+    int   selected; /**< Radio-button-id selected from the parent radio-button-list */
+    void* userData; /**< Associated user data */
+} tRadioButtonInfo;
+
+// SIMULATED DRIVING ASSISTANCE CHANGE: Added radiobutton-list information
+/// @brief Radio-button-list call-back information
+typedef struct RadioButtonListInfo
+{
+    int   amount;       /**< Number of radio buttons        */
+    int   dist;         /**< Distance between radio buttons */
+    int   selected;     /**< The radiobutton selected       */
+    bool  minimumOfOne; /**< If at least one radiobutton has to be selected at all times */
+    void* userData;     /**< Associated user data           */
```

Comment:
Public members, should follow the naming standards
```suggestion
    int   Amount;       /**< Number of radio buttons        */
    int   Dist;         /**< Distance between radio buttons */
    int   Selected;     /**< The radiobutton selected       */
    bool  MinimumOfOne; /**< If at least one radiobutton has to be selected at all times */
    void* UserData;     /**< Associated user data           */
```